specifyweb/frontend/js_src/lib/components/TreeView/Actions.tsx

```diff
@@ -77,7 +77,7 @@ export function TreeViewActions<SCHEMA extends AnyTree>({
         <li className="contents">
           {typeof focusedRow === 'object' ? (
             isRoot ? (
-              <Button.Small onClick={undefined}>
+              <Button.Small onClick={undefined} title={queryText.query()}>
```

Comment:
now that the button contains text again, there is no need for a title (especially since it's redundant)

I said earlier that title and aria-label is needed because the button was displaying only an icon without text

Reply:
same applies to all other button